matchbox_socket/src/integrations/bevy_plugin.rs

```diff
@@ -0,0 +1,57 @@
+use std::marker::PhantomData;
+
+use crate::{BuildablePlurality, WebRtcSocket, WebRtcSocketBuilder};
+use bevy::{ecs::system::Command, prelude::*, tasks::IoTaskPool};
+
+/// A [`Resource`] wrapping [`WebRtcSocket`].
+///
+/// To create and destroy this resource use the [`OpenSocket`] and [`CloseSocket`] [`Command`]s respectively.
+#[derive(Resource, Debug, Deref, DerefMut)]
+pub struct MatchboxSocket<C: BuildablePlurality>(pub WebRtcSocket<C>);
+
+/// A [`Command`] used to open a [`MatchboxSocket`] and allocate it as a resource.
+pub struct OpenSocket<C: BuildablePlurality>(WebRtcSocketBuilder<C>);
```

Comment:
could probably be `pub(crate)`?

Reply:
Depends if we want to allow people to use both `commands.open_socket` and `commands.add`, if not then yeah, `pub(crate)` would be better

Reply:
Have removed the visibility modifier all together since it's just one module and access only to the extension makes sense